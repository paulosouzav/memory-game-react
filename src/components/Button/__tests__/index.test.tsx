import React from 'react';
import { Button } from '../';
import { shallow } from 'enzyme';
import 'jest-styled-components';

describe('Button', () => {
  it('should render without failing', () => {
    const wrapper = shallow(<Button />);
    expect(wrapper).toMatchSnapshot();
  });
});
