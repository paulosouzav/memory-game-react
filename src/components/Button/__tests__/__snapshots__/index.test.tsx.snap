// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render without failing 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  border-radius: 8px;
  padding: 15px 35px;

  font-size: 18px;
  font-family: ",
          "Roboto",
          ";
  text-transform: uppercase;

  color: ",
          "#ffffff",
          ";
  background-color: ",
          "#6f42c1",
          ";
  border: 0;
  cursor: pointer;

  ",
          "
    box-shadow: 0 0 8px ",
          "rgba(0,0,0,0.5)",
          ";
  ",
          "

  transition: .5s background-color;
  &:hover {
    background-color: ",
          "#2d1a50",
          ";
  }
",
        ],
      },
      "displayName": "styled.button",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "button",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
